static/app/components/actions/archive.tsx

```diff
@@ -135,49 +134,47 @@ function ArchiveActions({
   });
 
   return (
-    <IssueActionWrapper>
```

Comment:
All I did here is remove the IssueActionWrapper which used to provide the animation